static/app/views/organizationGroupDetails/actions/shareIssue.tsx

```diff
@@ -73,7 +87,7 @@ function ShareIssue({loading, onReshare, onToggle, disabled, isShared, shareUrl}
         </ActionButton>
       }
       onOpen={handleOpen}
-      disabled={disabled}
+      disabled={disabled || !enabled}
```

Comment:
Imo, this is a tad confusing, disabled or not enabled. Imo, i'd move the enabled logic up into the parent component if possible